ports/servoshell/desktop/app.rs

```diff
@@ -430,11 +448,64 @@ impl App {
                         webview.go_forward(1);
                     }
                 },
-                WebDriverCommandMsg::SendKeys(..) |
-                WebDriverCommandMsg::KeyboardAction(..) |
-                WebDriverCommandMsg::MouseButtonAction(..) |
-                WebDriverCommandMsg::MouseMoveAction(..) |
-                WebDriverCommandMsg::WheelScrollAction(..) |
+                // Key events don't need hit test so can be forwarded to constellation for now
+                webdriver_msg @ WebDriverCommandMsg::SendKeys(..) => {
+                    running_state.forward_webdriver_command(webdriver_msg);
+                },
```

Comment:
You don't need to rename `msg` here. You can just use `msg` directly. This also applies to all other match patterns here:

```suggestion
               WebDriverCommandMsg::SendKeys(..) => {
                    running_state.forward_webdriver_command(msg);
                },
```